cmd/gen-stdlib-allowlists/main.go

```
@@ -0,0 +1,83 @@
+func parseFlags() *cliFlags {
+	flags := &cliFlags{}
+	flag.StringVar(&flags.goroot, "goroot", "", "Path to GOROOT (default: runtime.GOROOT())")
+	flag.StringVar(&flags.kind, "kind", "all", "Dataset: stutters | anyuse | sentinels | all")
+	flag.StringVar(&flags.outStutters, "out-stutters", "", "Output Go file for stutter pairs")
+	flag.StringVar(&flags.outAnyuse, "out-anyuse", "", "Output Go file for any-param functions")
+	flag.StringVar(&flags.outSentinels, "out-sentinels", "", "Output Go file for sentinel accessors")
+	flag.BoolVar(&flags.verify, "verify", false, "Emit to stdout; exit 2 if differs from on-disk file")
+	flag.Parse()
+	//: Default to build-host Go toolchain when --goroot is unset
+	if flags.goroot == "" {
+		flags.goroot = runtime.GOROOT()
```

golangci-lint (2.12.2)

[error] SA1019: runtime.GOROOT has been deprecated since Go 1.24: The root used during the Go build will not be meaningful if the binary is copied to another machine. Use the system path to locate the “go” binary, and use “go env GOROOT” to find its GOROOT. 

(staticcheck)